### ISSUE TITLE
Version woes

### ISSUE THREAD

**Issue by WolfgangWo** [JulianSauer/cb2java#1]
Hi!

TL;DR: Can you please help me spot a specific old version of cb2java in order for me to be able to start with a refactoring cycle? 

Recently. I've been given the task to take responsibility over a quite old program (say, ~2012) of our company the developer of which has long left. Moreover, the source code of one has been lost and all that is left is the jar file. However, JD-GUI is my friend.  I see that net.sf:cb2xml and net.sf:cb2java have been incorporated into the jar file. 

I'd like to start with the exact version of cb2java which apparently was used back then so that the project can be built. From there, I plan to start to update the libraries and make appropriate changes. Unfortunately, I have not yet been able to spot the version of cb2java that seems to have been used. 

It seems to me that a distinctive item that is present in the jar file is the class Decimal in package cb2java.copybook, not cb2java.copybook.types, that has a public method 

public boolean isPositiveSign(char last) {
    return isPositive(last);
 }

I have already tried versions 0.1 through 0.3.1 from https://sourceforge.net/projects/cb2java/ but did not find such a method.

By any chance, do you remember seeing such a method, having refotored it? I'd ver much appreciate your help.

Regards,
Wolfgang

**Comment by WolfgangWo** [comment]
In the meantime I've tried to grep(1) my way through all of the commits in this and in devstopfix's repository, and I didn't find any version which contains a public boolean isPositive*. Which makes the artifact I have at hand all the weirder ... :-/

If you have any idea?

**Comment by JulianSauer** [comment]
Hi Wolfgang,
I'm afraid I can't really help you with your problem. I forked the version from devstopfix that already didn't have that method.
Greetings
